documentation/latest/gsg/running_workloads.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: documentation
+title: Run workloads
+---
+
+Now that the devices are running, we need to run some container workloads on
+these devices, for that, the first thing that we need to do, it's to make sure
```

Comment:
s/these devices/them/ (?)
"... devices, for that, the first thing that we need to do"  ->  ". The first thing to do is to make sure"